src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -25,4 +36,31 @@ public ResponseEntity createLine(@RequestBody LineRequest lineRequest) {
         LineResponse line = lineService.saveLine(lineRequest);
         return ResponseEntity.created(URI.create("/lines/" + line.getId())).body(line);
     }
+
+    @GetMapping
+    public ResponseEntity showLines() {
```

Comment:
```suggestion
    public ResponseEntity<List<LineResponse>> showLines() {
```
모든 ResponseEntity에 제네릭 타입을 명시해주세요